src/onboarding/index.ts

```diff
@@ -0,0 +1,20 @@
+export interface OnboardingExperimentParams {
+  disableForcedBackup: boolean // Whether to disable forced 24h backup.
```

Comment:
nit: having this be a "disable" flag makes it a little harder to reason about imo. It's like a double negative-- to enable something, I need to not not enable it. I mention this because I think it makes bugs more likely. 

the only reason I'd want to keep it as a "disable" is if there was an existing feature flag that it was overriding. I don't think that's the case, though-- I think forced backup is hardcoded